### ISSUE TITLE
Migration/DepartmentName fails to autocorrect, but says it does

### ISSUE THREAD

**Issue by tas50** [rubocop-hq/rubocop#7619]
## Expected behavior

Running: `rubocop -a --only Migration/DepartmentName` should migrate legacy departments in rubocop comments.

## Actual behavior

Cop runs and shows that it autocorrected the department, but did not.

## Steps to reproduce the problem

Run rubocop against the followinng code:

```ruby
# encoding: UTF-8
# rubocop:disable SingleSpaceBeforeFirstArg, Metrics/LineLength
name             "apache_kafka"
```

## RuboCop version

```
0.78.0
```
